src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
```

Comment:
It is better to implement a getter for the ``result`` property.
This will ensure that the result value is accessed as a property and not as a variable.

Example:
```suggestion
function makeCalculator() {
  let result = 0;
  
  return {
```